Configuration.md

```diff
@@ -1,13 +1,34 @@
 # Configuring autopin+
 
-This documents describes how to configure ```autopin+```. The first two sections are specific to the configuration format implemented by the class ```StandardConfiguration```, which is used by default.
+This documents describes how to configure ```autopin+```. The first
+two sections are specific to the configuration format implemented by
```

Comment:
Isn't the first section specific to command-line options and not configuration format?